src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/MaskedBehavior.shared.cs

```diff
@@ -113,9 +118,9 @@ string RemoveMask(string text)
 			return string.Join(string.Empty, text.Split(maskChars));
 		}
 
-		void ApplyMask(string text)
+		void ApplyMask(string? text)
 		{
-			if (!string.IsNullOrWhiteSpace(text) && positions != null)
+			if (text != null && !string.IsNullOrWhiteSpace(text) && positions != null)
```

Comment:
Yea this is because Roslyn still thinks `text` could be null after `!string.IsNullOrWhiteSpace(text)`.

It's a bug in Roslyn, but using `text != null` is a solid enough work-around for now.